frontend/pages/admin/AppSettingsPage/cards/Agents/Agents.tsx

```diff
@@ -116,7 +114,7 @@ const Agents = ({
           </p>
           <YamlAce
             wrapperClassName={`${baseClass}__text-editor-wrapper`}
-            onChange={handleAceInputChange}
+            onChange={handleAgentOptionsChange}
```

Comment:
Minor code changes to global agent options code to match team agent options code